lisa/tools/lsof.py

```diff
@@ -18,6 +18,11 @@ def is_port_opened(self, port: int) -> bool:
         result = self.run(cmd, force_run=True, shell=True, sudo=True)
         return result.exit_code == 0
 
+    def is_port_opened_per_process_name(self, process_name: str) -> bool:
```

Comment:
I think it returns not only ports, also other file handlers. Can you confirm it?

Reply:
update.
```
2023-02-23 02:47:28.890[2616][DEBUG] lisa.env[generated_0].node[1].cmd[4596] cmd: ['sudo', 'sh', '-c', 'lsof -iTCP -P -n | grep -i lagscope'], cwd: None, shell: True, sudo: True, nohup: False, posix: True, remote: True
2023-02-23 02:47:28.890[4720][DEBUG] lisa.env[generated_0].node[1].cmd[6159].stdout lagscope 1.0.1
2023-02-23 02:47:28.890[4720][DEBUG] lisa.env[generated_0].node[1].cmd[6159].stdout ---------------------------------------------------------
2023-02-23 02:47:29.015[5376][DEBUG] lisa.env[generated_0].node[1].cmd[4596].stdout lagscope  11443            root    3u  IPv4  41189      0t0  TCP 10.0.0.4:6001 (LISTEN)
```